src/analytics/interface.test.js

```diff
@@ -89,6 +89,14 @@ describe('analytics identifyAuthenticatedUser', () => {
 });
 
 describe('analytics identifyAnonymousUser', () => {
+  beforeEach(() => {
```

Comment:
Probably don't need this to be in a `beforeEach`, right? Just needs to be defined once. Or maybe put up next to the other mocks (lines 45-48)